AjaxControlToolkit/Tabs/TabContainer.cs

```diff
@@ -427,8 +440,8 @@ public TabContainer()
 
         protected override void AddAttributesToRender(HtmlTextWriter writer) {
             Style.Remove(HtmlTextWriterStyle.Visibility);
-            if(!ControlStyleCreated)
-                writer.AddAttribute(HtmlTextWriterAttribute.Class, String.IsNullOrEmpty(CssClass) ? "ajax__tab_xp" : CssClass);
+
+            writer.AddAttribute(HtmlTextWriterAttribute.Class, GetCssClasses());
```

Comment:
What about writing a couple of tests for this? I mean testing markup rendered according to the `CssClass` and the `CssThemes` props values.